.clang-format

```diff
@@ -0,0 +1,119 @@
+# SPDX-FileCopyrightText: 2008-2021 HPDCS Group <rootsim@googlegroups.com>
+# SPDX-License-Identifier: CC0-1.0
+BasedOnStyle: LLVM
+AccessModifierOffset: -8
+AlignAfterOpenBracket: DontAlign
+AlignConsecutiveAssignments: None
+AlignConsecutiveBitFields: None
+AlignConsecutiveDeclarations: None
+AlignConsecutiveMacros: None
+AlignEscapedNewlines: Right
+AlignOperands: Align
+AlignTrailingComments: true
+AllowAllArgumentsOnNextLine: false
+AllowAllConstructorInitializersOnNextLine: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: Never
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortEnumsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLambdasOnASingleLine: All
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
```

Comment:
I'd set to `AllDefinitions`

Reply:
Consider it done!

Reply:
Actually, after inspection, setting `AlwaysBreakAfterReturnType` to `AllDefinitions` would conflict with the contributing docs.
Are you sure you want to change that?

Reply:
I wasn't able to find the part of the docs which would conflict with it, could you please give me a pointer?

Reply:
For example, in the contributing docs you can find this snippet:
```
void foo(void)
{
	...
}
```
With the change you proposed in the `AlwaysBreakAfterReturnType` option, the snippet would become:
```
void
foo(void)
{
	...
}
```

Reply:
I'm so dumb :smile:  I misread the doc, and thought this was affecting return statements rather than return types. My fault, please leave it as is!